evansmusomi/react-native-102
# Redirects to Find Places After Adding a Place and Shows All Added Places

*What this does*
- User is redirected to Find Places screen after sharing a place
- User always sees all places, including recently added, when they visit the Find Places screen

*How to verify*
- Share a place
- You should be redirected to the Find Places screen
- You should see the place you just added in the list of places
